webapi/_test_data/test_route/texts/beol/standoffToTEI.xsl

```diff
@@ -441,10 +441,22 @@
     http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-ref.html
     -->
     <xsl:template match="facsimile|figure">
-        <ref>
-            <xsl:attribute name="target"><xsl:value-of select="@src"/></xsl:attribute>
-            <xsl:apply-templates/>
-        </ref>
+        <xsl:choose>
+            <xsl:when test="ancestor::p">
+                <ref>
+                    <xsl:attribute name="target"><xsl:value-of select="@src"/></xsl:attribute>
+                    <xsl:apply-templates/>
+                </ref>
+            </xsl:when>
+            <xsl:otherwise>
+                <p>
+                    <ref>
+                        <xsl:attribute name="target"><xsl:value-of select="@src"/></xsl:attribute>
```

Comment:
@SepidehAlassi Once you have fixed the original XML, this is not necessary anymore.

Reply:
@tobiasschweizer In Goldbach letters, there were no errors about facsimiles links that did not occur inside a paragraph. There were cases of endnote references which were not in the paragraph, that I have fixed.

Reply:
@SepidehAlassi Great, so I will undo this hack :-)

Reply:
undone in https://github.com/dhlab-basel/Knora/pull/1260/commits/d0f85365ba67a98ec489a43cfead4229385760ba

Reply:
@tobiasschweizer I just checked this. I exported the TEI of the L0001 which has a facsimile. Validation of it fails because of the reference of facsimile outside of the paragraph. This is weird because this letter was not among the problematic xmls you had asked me to check before!
So sorry I have not noticed that there are actually references of facsimiles which should be wrapped in <p>
Now, either you have to leave the hack to be, or I have to change my xslt again to fix these. Then I would need to give you the new data again.